encoders/nlp/FarmTextEncoder/__init__.py

```diff
@@ -5,29 +5,31 @@
 
 class FarmTextEncoder(BaseTorchEncoder):
     """
+    Encode an array of string in size `B` into an ndarray in size `B x D`
+
     FARM-based text encoder: (Framework for Adapting Representation Models)
     https://github.com/deepset-ai/FARM
 
-    It encodes an array of string in size `B` into an ndarray in size `B x D`
+    :param model_name_or_path: Directory or public name of the model to load.
+    :param num_processes: Number of processes for `multiprocessing.Pool`.
+        Set to value of 0 to disable multiprocessing. Set to None to let
+        Inferencer use all CPU cores. If you want to debug the Language Model,
+        you might need to disable multiprocessing
+    :param extraction_strategy: Strategy to extract vectors. Choices:
+        'cls_token' (sentence vector), 'reduce_mean'(sentence vector),
+        reduce_max (sentence vector), 'per_token' (individual token vectors)
+    :param extraction_layer: number of layer from which the embeddings shall
+        be extracted. Default: -1 (very last layer).
+    :param args:  Additional positional arguments
+    :param kwargs: Additional keyword arguments
     """
 
     def __init__(self, pretrained_model_name_or_path: str = 'deepset/bert-base-cased-squad2',
                  num_processes: int = 0, extraction_strategy: str = 'cls_token',
                  extraction_layer: int = -1,
                  *args,
                  **kwargs):
-        """
-
-        :param model_name_or_path:  Local directory or public name of the model to load.
-        :param num_processes: the number of processes for `multiprocessing.Pool`. Set to value of 0 to disable
-                              multiprocessing. Set to None to let Inferencer use all CPU cores. If you want to
-                              debug the Language Model, you might need to disable multiprocessing!
-        :param extraction_strategy: Strategy to extract vectors. Choices: 'cls_token' (sentence vector), 'reduce_mean'
-                               (sentence vector), reduce_max (sentence vector), 'per_token' (individual token vectors)
-        :param extraction_layer: number of layer from which the embeddings shall be extracted. Default: -1 (very last layer).
-        :param args:
-        :param kwargs:
-        """
+        """Set Constructor."""
```

Comment:
Is this formulation recommended somewhere, `Set Constructor`? 

according to https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html it should just be 

```
        """Constructor method
        """
```

Reply:
I saw it in another example because it had to use imperative language

Reply:
@cristianmtr I think we agree upon the guidelines at https://docs.jina.ai/chapters/docstring/docstring.html#one-line-docstrings. 

Or else, we need to update our docstring guidelines.